### PR TITLE
Subject details

### PR DESCRIPTION
**Pull request by naid**

### CODE REVIEW

File: app/Http/Controllers/Supervisor/SubjectController.php [hunk 1]
```diff
@@ -18,31 +20,26 @@ public function __construct(SubjectRepositoryInterface $subjectRepository)
...
-        return view('supervisor.subjects.create');
-    }
+        $courses = $subject->course()->paginate(Course::COURSES_PER_PAGE);
 
-    public function show(Subject $subject)
-    {
-        return view('supervisor.subjects.show', [
-            'subject' => $this->subjectRepository->findOrFail($subject->id),
-            'tasks' => $subject->tasks,
+        return view('supervisor.subjects.show')->with([
```
Comment (nguyenthanhtung88) [inline app/Http/Controllers/Supervisor/SubjectController.php]: Your previous code is shorter, why you have to change to use `with`?

Comment (naid) [inline app/Http/Controllers/Supervisor/SubjectController.php]: sorry, this was during merge. reverting it back now

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -11,8 +11,9 @@
     <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css" rel='stylesheet' type='text/css'>
     <link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700" rel='stylesheet' type='text/css'>
 
-    <!-- Styles -->
+    <!-- Styles -->dsadas
```
Comment (nguyenthanhtung88) [inline resources/views/layouts/app.blade.php]: What is `dsadas` ???

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -11,8 +11,9 @@
...
     <link href="{{ url('css/app.css') }}" rel="stylesheet">
+    @import "path/to/css/app.scss";
```
Comment (nguyenthanhtung88) [inline resources/views/layouts/app.blade.php]: Did it work ???

File: resources/views/supervisor/subjects/index.blade.php [hunk 4]
```diff
@@ -35,19 +49,58 @@
...
+                                                {!! Form::hidden('subject_id',$subject->id) !!}
+                                                {!! Form::button('<i class="fa fa-btn fa-trash-o"></i>',[
+                                                        'class' => 'btn btn-danger',
+                                                        'type' => 'submit'
+                                                    ])
+                                                !!}
+                                            {!! Form::close() !!}
+                                        </div>
+                                        <div class="col-md-2">
+                                            {!! Form::open(['method' => 'get', 'route' => [
```
Comment (nguyenthanhtung88) [inline resources/views/supervisor/subjects/index.blade.php]: Default form open is `GET` method?
I saw in this situation, you only need like to edit subject right?
Make a link i think it's enough

File: resources/views/supervisor/subjects/index.blade.php [hunk 5]
```diff
@@ -35,19 +49,58 @@
...
+                                                {!! Form::hidden('subject_id',$subject->id) !!}
+                                                {!! Form::button('<i class="fa fa-btn fa-pencil"></i>',[
+                                                        'class' => 'btn btn-warning',
+                                                        'type' => 'submit'
+                                                    ])
+                                                !!}
+                                            {!! Form::close() !!}
+                                        </div>
+                                        <div class="col-md-1">
+                                            {!! Form::open(['method' => 'get', 'route' => [
```
Comment (nguyenthanhtung88) [inline resources/views/supervisor/subjects/index.blade.php]: Make link instead of open form get

**Comment by nguyenthanhtung88** [comment]
OK

**Pull request merged**